frontend/cypress/integration/local.spec.ts

```diff
@@ -0,0 +1,5 @@
+describe('Local server', () => {
+  it('successfully loads', () => {
+    cy.visit('/')
```

Comment:
This is the actual first test case. It just visit the page and see if 200 is returned.